skycoin/skywire
# improve logging

Did you run `make format && make check`? Yes

Fixes #1472 #1473 #1427

 Changes:	
- [x] add `-l` or `--store-log` flag to *skywire-visor* for save logs in `/local/log/...` path as `{date}-{start_time}.log` file.
- [x] add `--force-color` flag to *skywire-visor* for set `ForceColor` attribute of formatter true on non-TTY outputs.
- [x] fix missing logs in `/runtime-logs` api endpoint
- [x] fix app logs wrong response is `/logs` endpoint of apps
- [x] fix since query filter on `/logs` 

How to test this PR:
1. run visor as `skywire-visor -c skywire-config.json -l`, then checking `/local/log/` folder.
2. run visor on non-TTY output with `--force-color` flag as `skywire-visor -c skywire-config.json --force-color`
3. run visor and check logs of visor and specific app like skychat or skysocks
